src/sentry/snuba/snuba_query_validator.py

```diff
@@ -95,6 +95,12 @@ class Meta:
 
     data_source_type_handler = QuerySubscriptionDataSourceHandler
 
+    def __init__(self, *args, timeWindowSeconds=False, **kwargs):
+        super().__init__(*args, **kwargs)
+        # if true, time_window is interpreted as seconds.
+        # if false, time_window is interpreted as minutes.
+        self.time_window_seconds = timeWindowSeconds
```

Comment:
When do we expect to be using this as minute versus seconds? Can it just be one or the other?